test/topology_random_failures/suite.yaml

```diff
@@ -0,0 +1,11 @@
+type: Topology
```

Comment:
Do we need another suite? Can't we reuse `topology_custom`? It also gives a 0 size cluster.

Reply:
Yes, it's better to use a separate suite here because of own conftest and some python modules which are for this test only.